src/fmu/dataio/dataio.py

```diff
@@ -931,11 +935,12 @@ class AggregatedData:  # pylint: disable=too-few-public-methods
             Default is None which means it will be missing (null) in the metadata.
     """
 
+    aggregation_id: Optional[Union[str, bool]] = None
+    casepath: Union[str, Path, None] = None
     configs: list = field(default_factory=list)
```

Comment:
Should we consider changing `configs` into something more descriptive, e.g. `source_metadatas` or `input_metadatas` or something. Not sure, but I think "config" is confusing.

Reply:
Good point, I change it to `source_metadata` (not "datas", since data already is plural)